src/rules/heights.ts

```diff
@@ -1,30 +1,45 @@
-import { RuleFn } from "./RuleConfig";
-import { makeIncRules, makeRules } from "../utils";
+import { RuleFn } from './RuleConfig';
```

Comment:
This should still be `"`?

Reply:
ah vscode and prettier auto saved those as single quotes. I'll revert back to double